pre-provisioned/run-performance-tests.sh

```diff
@@ -31,178 +31,6 @@ db_password="wso2carbon"
 # Execute common script
 . $script_dir/perf-test-pre-provisioned.sh
 
-declare -A test_scenario0=(
```

Comment:
For the pre-provisioned environment shall we keep it as it is, since we haven't introduced other changes also to the pre-provisioned environment. Before updating the pre-provisioned environment related files, better to do a dry run and check.

Reply:
0079f50b1987595f2f43d874b2d5cd2b11a4fa35